WooCommerce/Classes/ViewRelated/CardPresentPayments/CardPresentPaymentsModalViewController.swift

```diff
@@ -188,12 +188,14 @@ private extension CardPresentPaymentsModalViewController {
         primaryButton.applyPrimaryButtonStyle()
         primaryButton.titleLabel?.adjustsFontSizeToFitWidth = true
         primaryButton.titleLabel?.minimumScaleFactor = 0.5
+        primaryButton.titleLabel?.lineBreakMode = .byClipping
```

Comment:
TIL `.byClipping`. I definitely had trouble with moving to the new ButtonConfiguration API.